pages/data/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="../../styles/reset.css">
+  <link rel="stylesheet" href="../../styles/style.css">
+  <link rel="stylesheet" href="./cart.css">
+  <title>Cart</title>
+</head>
+
+<body>
+  <header>
+    <h1>Frog Hats!</h1>
+    <nav>
+      <ul>
+        <li>
+          <a href="/Products/product.js">
+            Products
+          </a>
+        </li>
+        <li>
+          <a href="../../">Home</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="cart--container">
+    <h2>Shopping Cart</h2>
+    <table class="cart--table">
+      <thead>
+        <tr>
+          <th>
+            Product Name
+          </th>
+          <th>
+            Quantity
+          </th>
+          <th>
+            Price
+          </th>
+          <th>
+            SubTotal
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+      </tbody>
+      <tfoot>
+      </tfoot>
+    </table>
+    <button class="order--button">Place Order!</button>
+  </main>
+  <footer>
+
+  </footer>
+
+  <script type="module" src="cart.js"></script>
```

Comment:
pathing: currently this should be 
  <script type="module" src="../../pages/cart.js"></script>